tutorials/HDF5/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -0,0 +1,241 @@
+---
+title: "Subsetting HDF5 files to make a smaller deliverable"
+code1: HDF5/Subset-HDF5-hyperspectral.R
+contributors: null
+dataProudct: null
+dateCreated: 2020-02-04 14:20:52
+description: Take a large HDF5 file and extract only the information that you need,
+  then save as a new HDF5 file. For an example, we will take an existing hyperspectral
+  dataset (~600Mb) and reduce it in size for subsequent tutorials.
+estimatedTime: 1.0 Hours
+languagesTool: R
+packagesLibraries: rhdf5, raster
+syncID: 884cc5915a3842c1ab6617b060797fab
+authors: Donal O'Leary
+topics: hyperspectral, HDF5, remote-sensing
+tutorialSeries: null
+urlTitle: subset-hdf5-r
+---
+
+In this tutorial, we will subset an existing HDF5 file containing NEON
+hyperspectral data. The purpose of this exercise is to generate a smaller
+file for use in subsequent tuorials.
+
+<div id="ds-objectives" markdown="1">
+
+## Learning Objectives
+
+After completing this activity, you will be able to:
+
+* Navigate an HDF5 file to identify the variables of interest.</li>
+* Generate a new HDF5 file from a subset of the existing dataset.</li>
+* Save the new HDF5 file for future use.
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
+
+### R Libraries to Install:
+
+* **rhdf5**: `install.packages("BiocManager")`, `BiocManager::install("rhdf5")`
+* **raster**: `install.packages('raster')`
+
+<a href="{{ site.baseurl }}/packages-in-r" target="_blank"> More on Packages in
+ R - Adapted from Software Carpentry.</a>
+
+### Data to Download
+The purpose of this tutorial is to reduce a large file (~`652Mb`) to a smaller size. The download linked here is the original large file, and therefore you may choose to skip this tutorial and download if you are on a slow internet connection or have file size limitations on your device.
+
+{% include/dataSubsets/_data_Imaging-Spec-Data-Full-Tile-H5-2020.html %}
+
+This file is NEON_D17_SJER_DP3_257000_4112000_reflectance.h5 from March 2019, available on data.neonscience.org.
+
+***
+{% include/_greyBox-wd-rscript.html %}
+***
+
+### Recommended Skills
+
+If you haven't already, we highly recommend you work through the 
+<a href="{{ site.baseurl }}/intro-hdf5-r-series" target="_blank"> *Introduction to Working with HDF5 Format in R* series</a>
+before moving on to this tutorial.
```

Comment:
Good point. I added some context here, so that now it reads:
"For this tutorial, we recommend that you have some basic familiarity with the HDF5 file format, including knowing how to open HDF5 files (in Rstudio or HDF5Viewer) and how groups and metadata are structured. To brush up on these skills, we suggest that you work through the <a href="{{ site.baseurl }}/intro-hdf5-r-series" target="_blank"> *Introduction to Working with HDF5 Format in R* series</a> before moving on to this tutorial."